src/safeposix/dispatcher.rs

```diff
@@ -595,18 +617,24 @@ pub extern "C" fn lindrustinit(verbosity: isize) {
         sem_table: interface::RustHashMap::new(),
     };
     interface::cagetable_insert(1, initcage);
+    // make sure /tmp is clean
+    let init:bool = true;
+    cleartmp(init);
```

Comment:
```suggestion
    cleartmp(true);
```
Do we need to create a new variable?